pinakes/main/approval/models.py

```diff
@@ -45,19 +46,16 @@ class SettingField(models.TextField):
     """Customized field to store encrypted settings"""
 
     def get_prep_value(self, value):
-        """override"""
```

Comment:
While I agree that comments that don't add much value should be removed, I don't know if it's relevant to this PR to remove them; in the future we should probably keep small code cleanups like this to separate PRs. Since they are trivial, though, I think it's fine to leave them here and if you can advise on the specifics of the pylint rule to be amended and how, that would be great.